### PR TITLE
Chapters page implementation

### PR DESCRIPTION
**Pull request by temilolakutelu**

### CODE REVIEW

File: templates/views/chapters.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            </li>
+            {% endfor %}
+        </ul>
+    </div>
+
+    <div class="row tab-content">
+        {% for member in chapters %}
+        <div class="col-lg-2 col-md-3 col-sm-4 col-xs-12 filter {{member.categories}}">
+            <div class="box">
+                <h5>{{member.slug}}</h5>
```
Comment (OdunayoOkebunmi) [inline templates/views/chapters.html:46]: Should it be the `slug` or the `name` that should be displayed?

File: templates/views/chapters.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            <h2>She Code Africa <br> Chapters</h2>
+        </div>
+        <div class="banner-subtitle">
+            <p>
+                You might want to consider starting a SheCodeAfrica Chapter in your campus or city. Not an expert in tech yet? Not a problem (CTA)
+            </p>
+        </div>
+        <div class="banner-btn">
+            <button class="btn sca-btn pink-btn">JOIN A CHAPTER</button>
+            <button class="btn sca-btn trans-btn">LEAD A CHAPTER</button>
```
Comment (OdunayoOkebunmi) [inline templates/views/chapters.html:15]: On `:focus`, the text becomes less visible
Screenshot 2020-09-03 at 10 22 22

**Review by OdunayoOkebunmi — CHANGES_REQUESTED** [review]

File: templates/views/chapters.html (hunk 2 above)
Comment (temilolakutelu) [reply templates/views/chapters.html:15]: I  didn't experience this, can you refresh?

Comment (OdunayoOkebunmi) [reply templates/views/chapters.html:15]: Still the same. It happens when I click on the button

**Review by OdunayoOkebunmi — APPROVED** [review]

**Pull request merged**